data/json/recipes/other/materials.json

```diff
@@ -32,8 +32,7 @@
       [ [ "forge", 80 ], [ "char_forge", 80 ] ]
     ],
     "components": [
```

Comment:
That almost sounds more tedious and complicated that fixing the formatting personally, if it can't simply be done by running the tool through command line.